src/app/components/EnterPage/enter.scss

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Mulish');
+
+:local {
+  .enter {
+    position: fixed;
+    width: 426px;
+    height: 238px;
+    left: calc(747/1920*100%);
+    top: 96px;
```

Comment:
Don't use `position: fixed` for this simple component.
Just use the default value (`position: static` or simply unset) and use `margin-top` instead of `top`, use `left: 50%` and `transform: translateX(-50%)` instead of `left: calc(...)`